test/logger_test.cc

```diff
@@ -2,6 +2,7 @@
 #include "gtest/gtest.h"
 
 TEST(TestLogger, DummyTest) {
-  BOOST_LOG_TRIVIAL(trace) << "A trace severity message";
+  blsb::LoggerType local_logger;
```

Comment:
@AGaliuzov, please description of the ticket. I'm planning to have two macroses:
1) with logger LOG_TRACE*L*(logger)
2) second one will rely on existing in the scope method\function `LogerType& getLogger()`
and reuse the first one.
```
TRACE_LOG() 
auto& logger = getLogger()
LOG_TRACE*L*(logger)
```